advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,591 @@
+
+$ pgbackrest info --stanza=demo --repo=2
+stanza: demo
+    status: ok
+    cipher: none
+
+    db (current)
+        wal archive min/max (13): 00000001000000000000000E/000000010000000000000012
+
+        full backup: 20210419-142414F
+            timestamp start/stop: 2021-04-19 14:24:14 / 2021-04-19 14:24:26
+            wal start/stop: 00000001000000000000000E / 00000001000000000000000E
+            database size: 50.5MB, database backup size: 50.5MB
+            repo2: backup set size: 8MB, backup size: 8MB
+
+        incr backup: 20210419-142414F_20210419-142556I
+            timestamp start/stop: 2021-04-19 14:25:56 / 2021-04-19 14:25:57
+            wal start/stop: 000000010000000000000012 / 000000010000000000000012
+            database size: 50.6MB, database backup size: 661.5KB
+            repo2: backup set size: 8MB, backup size: 28KB
+            backup reference list: 20210419-142414F
+```
+
+The **'wal archive min/max'** shows the minimum and maximum WAL currently stored in the archive and, in the case of multiple repositories, will be reported across all repositories unless the `--repo` option is set. Note that there may be gaps due to archive retention policies or other reasons.
+
+Let us break our first repository by removing its content:
+
+```bash
+$ pgbackrest --stanza=demo stanza-delete --repo=1 --force
+P00   INFO: stanza-delete command end: completed successfully
+
+$ pgbackrest --stanza=demo stanza-create
+P00   INFO: stanza-create for stanza 'demo' on repo1
+P00   INFO: stanza-create for stanza 'demo' on repo2
+P00   INFO: stanza 'demo' already exists on repo2 and is valid
+P00   INFO: stanza-create command end: completed successfully
+```
+
+If there are multiple repositories, then a status of **mixed** indicates that the stanza is not in a healthy state in one or more of the repositories. In this case, the state of the stanza will be detailed in additional lines per repository. 
+
+```bash
+$ pgbackrest info --stanza=demo
+stanza: demo
+    status: mixed
+        repo1: error (no valid backups)
+        repo2: ok
+    cipher: none
+
+    db (current)
+        wal archive min/max (13): 00000001000000000000000E/000000010000000000000012
+
+        full backup: 20210419-142414F
+            timestamp start/stop: 2021-04-19 14:24:14 / 2021-04-19 14:24:26
+            wal start/stop: 00000001000000000000000E / 00000001000000000000000E
+            database size: 50.5MB, database backup size: 50.5MB
+            repo2: backup set size: 8MB, backup size: 8MB
+
+        incr backup: 20210419-142414F_20210419-142556I
+            timestamp start/stop: 2021-04-19 14:25:56 / 2021-04-19 14:25:57
+            wal start/stop: 000000010000000000000012 / 000000010000000000000012
+            database size: 50.6MB, database backup size: 661.5KB
+            repo2: backup set size: 8MB, backup size: 28KB
+            backup reference list: 20210419-142414F
+```
+
+Fix it by taking a new backup:
```

Comment:
This state can be resolved by taking a new backup.

Reply:
Fixed in 7b5477c